### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by aringocode**
### Changes
1. version bump
2. update Changelog

### CODE REVIEW

**Comment by aringocode** [comment]
@Pe5h4 Can you review pls?

**Review by Pe5h4 — COMMENTED** [review]
@aringocode I thought you will use the help component in SeaCat Admin UI - meaning you will pass the URL to the https://docs.teskalabs.com/seacat-auth/

**Comment by aringocode** [comment]
@Pe5h4 

> @aringocode I thought you will use the help component in SeaCat Admin UI - meaning you will pass the URL to the https://docs.teskalabs.com/seacat-auth/

I thought I would add to it when the documentation is complete

**Comment by Pe5h4** [comment]
> @Pe5h4
> 
> > @aringocode I thought you will use the help component in SeaCat Admin UI - meaning you will pass the URL to the https://docs.teskalabs.com/seacat-auth/
> 
> I thought I would add to it when the documentation is complete

@aringocode Alright then, but please stay in touch with Robin about it

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
-- ASAB WebUI submodule version bump [[4675693](https://github.com/TeskaLabs/asab-webui/commit/46756931bf295b40d00cf23ab8849233c35fed24)] commit (INDIGO Sprint 230623, [!46](https://github.com/TeskaLabs/seacat-admin-webui/pull/46))
+- ASAB WebUI submodule version bump [[0a1b9ce](https://github.com/TeskaLabs/asab-webui/commit/0a1b9ceb6ab7dfafd0cc9cf44459e7f1b4e1f2ca)] commit (INDIGO Sprint 230623, [!48](https://github.com/TeskaLabs/seacat-admin-webui/pull/48))
 
 ### Refactoring
 
 - Add styles for primary dropdown-toggle (INDIGO Sprint 230623, [!428](https://github.com/TeskaLabs/asab-webui/pull/428))
 
+### Bugfix
+
+- Refactor HelpComponent, remove HelpService (INDIGO Sprint 230609, [!418](https://github.com/TeskaLabs/asab-webui/pull/418))
```
Comment (Pe5h4) [inline CHANGELOG.md:15]: @aringocode This should be below Refactoring section :)

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**